checker/tests/initialization/fbc/ReceiverSuperInvocation.java

```diff
@@ -0,0 +1,16 @@
+// Test case for issue 2263
+// https://github.com/typetools/checker-framework/issues/2263
+
+import org.checkerframework.checker.initialization.qual.UnderInitialization;
+
+class ReceiverSuperInvocation {
+    void foo(@UnderInitialization(ReceiverSuperInvocation.class) ReceiverSuperInvocation this) {}
+}
+
+class Sub extends ReceiverSuperInvocation {
```

Comment:
One slight nit I just notice: `Sub` is a bad name for this class as it might easily cause a clash with another test case that also uses `Sub`.
In general we should either keep to one top-level class or use unique names. So as a small cleanup we should maybe rename this to `ReceiverSuperInvocationSub` or some such.